crates/interledger-service-util/src/exchange_rate_providers/cryptocompare.rs

```diff
@@ -79,7 +79,8 @@ pub fn query_cryptocompare(
             let rates = res
                 .data
                 .into_iter()
-                .map(|asset| (asset.coin_info.name.to_uppercase(), asset.raw.usd.price))
+                .filter(|asset| asset.raw.is_some())
+                .map(|asset| (asset.coin_info.name.to_uppercase(), asset.raw.unwrap().usd.price))
```

Comment:
Unwrapping will cause the node to panic. :-1: 

```suggestion
                .filter_map(|asset| if let Some(raw) = asset.raw {
                  Some((asset.coin_info.name.to_uppercase(), raw.usd.price))
                } else {
                  None
                })
```

Reply:
Even if the assets are filtered by `is_some`?
Is that possible?

Reply:
No, you're right. I didn't see that. Could you change it anyway? I think it's clearer to do it in one step instead of two

Reply:
Pushed a change that addresses this in https://github.com/interledger-rs/interledger-rs/pull/503/commits/b55a71935124628d098387cdf7b44da3772f57d7